<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>JakaGUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/Black★RockShooter.ico</normaloff>:/Black★RockShooter.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
	background-image: url(:/background.webp);
}
#addr_lineEdit{
	padding-left:20px;
	background-color: rgb(255, 255, 255);
	border:2px solid rgb(20,196,188);
	border-radius:15px;
}
#com_lineEdit{
	padding-left:20px;
	background-color: rgb(255, 255, 255);
	border:2px solid rgb(20,196,188);
	border-radius:15px;
}
#addr_label{
	background-color: rgba(0, 0, 0, 0);
	font: 10pt &quot;微软雅黑&quot;;
}
#com_label{
	background-color: rgba(0, 0, 0, 0);
	font: 10pt &quot;微软雅黑&quot;;
}
#config_label{
	background-color: rgba(0, 0, 0, 0);
	font: 10pt &quot;微软雅黑&quot;;
}
#recv_label{
	background-color: rgba(0, 0, 0, 0);
	font: 10pt &quot;微软雅黑&quot;;
}
#getID_label{
	background-color: rgba(0, 0, 0, 0);
	font: 10pt &quot;微软雅黑&quot;;
}
#server_label{
	background-color: rgba(0, 0, 0, 0);
	font: 10pt &quot;微软雅黑&quot;;
}
#status_label{
	background-color: rgba(0, 0, 0, 0);
	font: 10pt &quot;微软雅黑&quot;;
}
#status_button{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}
#ready_button{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}
#ready_button:pressed{
	color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}
#ready_button:hover{
	color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}
#server_switch{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}
#server_switch:hover{
	color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}
#server_switch:pressed{
	color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}
#getID_switch{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}
#getID_switch:hover{
	color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}
#getID_switch:pressed{
	color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}
#recv_switch{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}
#recv_switch:hover{
	color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}
#recv_switch:pressed{
	color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}
#clear_button{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}
#clear_button:hover{
	color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}
#clear_button:pressed{
	color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}
#send_button{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}
#send_button:hover{
	color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}
#send_button:pressed{
	color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}
#deepLearning_start{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}
#deepLearning_start:hover{
	color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}
#deepLearning_start:pressed{
	color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}
#deepLearning_open{
	font: 10pt &quot;微软雅黑&quot;;
    color: #2f3640;
    background-color: #f5f6fa;
    border-color: #2f3640;
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    padding: 5px;
}
#deepLearning_open:hover{
	color: #FFFFFF;
    background-color: #718093;
    border-color: #2f3640;
}
#deepLearning_open:pressed{
	color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #273c75, stop:1 #487eb0);
}
#send_lineEdit{
	padding-left:20px;
	background-color: rgb(255, 255, 255);
	border:2px solid rgb(20,196,188);
	border-radius:15px;
}
#meso_textBrowser{
	border-image: url(:/sensei.webp);
}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>128</width>
    <height>128</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="addr_lineEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="com_lineEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>12345</string>
    </property>
   </widget>
   <widget class="QLabel" name="addr_label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>地址</string>
    </property>
   </widget>
   <widget class="QLabel" name="com_label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>端口</string>
    </property>
   </widget>
   <widget class="QLabel" name="config_label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>参数设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ready_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CONFIRM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="server_switch">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getID_switch">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recv_switch">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QLabel" name="server_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>服务端</string>
    </property>
   </widget>
   <widget class="QLabel" name="getID_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>监听</string>
    </property>
   </widget>
   <widget class="QLabel" name="recv_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>接收</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="meso_textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>180</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>470</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>CLEAR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="send_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="send_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>SEND</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deepLearning_start">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>dSTART</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deepLearning_open">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>dOPEN</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
